File: frontend/app/src/main/java/com/example/caffwebshop/model/UserRegistrationModel.kt
package com.example.caffwebshop.model

import com.google.gson.annotations.SerializedName

data class UserRegistrationModel (
    @SerializedName("userName")
    val userName:Int,
    @SerializedName("email")
    val email: String,
    @SerializedName("firstName")
    val firstName: String,
    @SerializedName("lastName")
    val lastName: String,
    @SerializedName("password")
    val password: String

)

File: backend/CaffShop/Models/User/UserRegistrationModel.cs
using System.ComponentModel.DataAnnotations;
using System.Diagnostics.CodeAnalysis;

namespace CaffShop.Models.User
{
    [SuppressMessage("ReSharper", "UnusedAutoPropertyAccessor.Global")]
    [SuppressMessage("ReSharper", "UnusedMember.Global")]
    public class UserRegistrationModel
    {
        [Required]
        public string UserName { get; set; }

        [Required]
        public string Email { get; set; }
        
        [Required]
        public string FirstName { get; set; }

        [Required]
        public string LastName { get; set; }
        
        [Required]
        public string Password { get; set; }
    }
}